src/main/resources/config/config.yaml

```diff
@@ -73,6 +75,7 @@ graphitePort: ${METRICS_PORT:-8092}
 
 links:
   selfserviceUrl: ${SELFSERVICE_URL}
+  frontendUrl: ${FRONTEND_URL}
```

Comment:
I would build all the necessary links here rather than using constants with path into the app as I said before